viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -32,30 +27,22 @@ struct ColorARGB {
     unsigned char B;
 };
 
-// Check if pixel is the default color signaling no data is present
-bool CheckIfEmptyPixel(ColorARGB pixel_color) {
-    return (pixel_color.R == defaultNoDataRGBValue) &&
-           (pixel_color.G == defaultNoDataRGBValue) &&
-           (pixel_color.B == defaultNoDataRGBValue);
-}
+// Check the green color channel to filter unobserved pixels
```

Comment:
```suggestion
// Check if the green color channel is 0 to filter unobserved pixels
```

Reply:
should the comment mention where unobserved pixels comes from? i.e. linking to https://github.com/cartographer-project/cartographer/blob/master/cartographer/io/submap_painter.cc#L210-L213

Reply:
yes please!

Reply:
added!